componentLibraries/extensionLibrary/extensionLibrary.xml

```diff
@@ -28,6 +28,8 @@
   <component>Hydraulic/HydraulicVariableVolumeLength.hpp</component>
   <component>Hydraulic/HydraulicVariableVolumeLengthMP.hpp</component>
   <component>Electrical/ElectricInductanceC.hpp</component>
+  <component>Connectivity/FMIWrapper.hpp</component>
+  <component>Signal/FibonacciNumbers.hpp</component>
```

Comment:
I guess Fibanatcci was some component that were already present or is it by mistake ?

Reply:
Right, my bad. I made that component first only to test the reconfiguration stuff with something simple.

Reply:
Should the file be removed from this list?
The file it self was not added

Reply:
Right. I missed it since the library XML file is not in the Qt project, so search function did not find it. Maybe we should add all library/component XML files as auxiliary files?